section3/exercises/loops-arrays.js

```diff
@@ -42,8 +56,15 @@ var names = ["Sevy", "Lindsey", "Jericho", "Raeonna", "Latrina", "Jhana", "Luca"
   // 18 or over? "Make sure you get out and vote!"
 
 // WORD FROM THE WISE: Before you start coding, jot down, with pen and paper, what steps you
-// want to tell the program to take. Think of all the concepts you've learned so far. 
+// want to tell the program to take. Think of all the concepts you've learned so far.
 
 // Remember, stuck for 15-20 minutes? Reach out to your Mod 0 cohort on Slack.
 
-var ages = [17, 20, 34, 18, 16, 65, 49, 10, 22, 18];
\ No newline at end of file
+var ages = [17, 20, 34, 18, 16, 65, 49, 10, 22, 18];
+for (var i = 0; i < ages.length; i++){
+  if ( ages[i] < 18) {
+    console.log("Oh no, you can't vote quite yet.");
+  } else if ( ages[i] >= 18){
+    console.log("Make sure you get out and vote!");
+  }
+}
```

Comment:
Nice, you have a good understanding of looping through arrays. Just want to see you clean up your styling conventions going forward (lots of whitespace out of place or missing)